natives.json

```diff
@@ -108989,12 +108989,12 @@
 			"comment": "Example : https://pastebin.com/h1YzycuR",
 			"params": [
 				{
-					"type": "Any*",
-					"name": "p0"
+					"type": "int*",
```

Comment:
This should stay ``Any*`` as it's a pointer to a struct (where the first field is the duration).